src/css/Home.css

```diff
@@ -0,0 +1,75 @@
+.Home,
+.JoinList-container {
```

Comment:
We ended up including the CSS of the joinlist page because we realized it was pulling from Home.css even though it wasn't imported. 

Actually, all of the .css files were being imported throughout the app, so not sure if we really need a specific .css folder or just one massive App.css??

Reply:
Might be good to have one central CSS file for sure!